python/src/CombinationsDistribution_doc.i.in

```diff
@@ -0,0 +1,84 @@
+%feature("docstring") OT::CombinationsDistribution
+"Combinations distribution.
+
+Parameters
+----------
+k : int, :math:`k > 0`
+n : int, :math:`n > 0`
+
+See also
+--------
+Combinations
+
+Notes
+-----
+:class:`~openturns.CombinationsDistribution` is the discrete uniform
+distribution on the set of strictly increasing functions :math:`(i_0, \hdots, i_{k_1})`
+from :math:`\{0, \hdots, k-1\}` into :math:`\{0, \hdots, n-1\}`.
+Its probability distribution function is defined as:
+
+.. math::
+
+    \Prob{\vect{X} = (i_0, \hdots, i_{k-1})} = \frac{1}{\binom{n}{k}}
+
+Its first moments are:
+
+.. math::
+    :nowrap:
+
+    \begin{eqnarray*}
```

Comment:
These formulas are a copy/paste from KPermutationsDistribution, Anne has to change it into my branch soon